runtime/proto/log_unit.proto

```diff
@@ -0,0 +1,150 @@
+syntax = "proto3";
```

Comment:
By "all externally facing API messages", do you mean those that correspond directly to an RPC request/response type?

Reply:
Yep.

Reply:
Hi @vjeko , including all the API based messages into a new file called `service.proto` and all their dependencies into a `helper.proto` file would again make these 2 new files very big (approximately half of the size of `CorfuProtocol.proto` file).

Did you mean clustering/moving the current API layer-based files like `log_unit.proto`, `sequencer.proto`, etc into a new folder `runtime/proto/service` and package `org.corfudb.runtime.proto.service`? This makes a lot of sense but it would leave out only non-API ones out. Should we move them into a `runtime/proto/common` folder or leave them as is outside in `runtime/proto`?

Reply:
I was referring to the later approach. I would personally leave them in `runtime/proto`. Generally, concepts like `common` should be avoided: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Did my best to restructure while keeping this in mind. Let me know if there is any other feedback